india_compliance/tests/test_records.json

```diff
@@ -344,6 +344,26 @@
                     "link_name": "_Test Registered Composition Supplier"
                 }
             ]
+        },
+        {
+            "name": "_Test Indian Registered Company-Shipping",
+            "address_type": "Shipping",
+            "address_line1": "Test Address - 5",
+            "city": "Test City",
+            "state": "Gujarat",
+            "pincode": "380015",
+            "country": "India",
+            "gstin": "24AAQCA8719H1ZC",
+            "gst_category": "Registered Regular",
+            "is_primary_address": 1,
+            "is_shipping_address": 1,
```

Comment:
If it's not set to primary or default shipping, you would not have required to make changes to test data files with the company address.

Reply:
I have added this to test 'ShipDtls' in e_invoice test case.